client/src/components/attendance/AttendanceForm.js

```diff
@@ -276,12 +277,27 @@ export default function AttendanceForm(props) {
         renderCell: (params) => (
           <Slider className={classes.slider} onChangeCommitted={(e, value) => handleCapacityChange(value, params)}
                   disabled={params.getValue('isAbsent')}
-                  step={1} min={1} max={4} defaultValue={params.getValue('capacity')} data-capacity={params.getValue('capacity')} />
+                  step={1} min={1} max={4} defaultValue={params.getValue('capacity')}
+                  data-capacity={params.getValue('capacity')}/>
         )
       }
     )
   }
 
+  let history = useHistory()
```

Comment:
Put const here for `const history....` as history reference cannot be changed.

Also move this to the top of the file (just under `const classes...`)